src/main/scala/devices/debug/Debug.scala

```diff
@@ -1115,6 +1371,9 @@ class ClockedDMIIO(implicit val p: Parameters) extends ParameterizedBundle()(p){
 
 class DMIToTL(implicit p: Parameters) extends LazyModule {
 
+  import uncore.devices._
+  import uncore.devices.DMI_RegAddrs._
+
```

Comment:
did you actually need to add this import here?

Reply:
Yeah, the compiler complained that it couldn't find "DMI_RegAddrs.DMI_CONTROL", which is strange because this was working with the previous version of dm_registers.scala

Reply:
I think that is because you need to manually change the package name in the `dm_registers.scala`